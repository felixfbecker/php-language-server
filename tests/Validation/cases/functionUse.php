<?php

$a = new A;

$b = function () use ($a) {
    echo $a->b();
};