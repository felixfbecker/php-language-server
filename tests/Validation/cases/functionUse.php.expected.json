{
    "references": {
        "A": [
            "./functionUse.php"
        ],
        "A->b()": [
            "./functionUse.php"
        ]
    },
    "definitions": []
}